### DISCUSSION TITLE
Section 8 Transfer Learning: Do I need to know all the functions used in a pre-trained model like these?

### DISCUSSION THREAD

**Discussion by mosaabthemuslim** [mrdbourke/pytorch-deep-learning#985] [Q&A]
Like batchnorm2d and MBconv.
What to do?
thanks
![342108565-96c6412c-a447-440f-a06c-bfe9deacc940](https://github.com/mrdbourke/pytorch-deep-learning/assets/127981674/e1ec90c8-3959-44e3-b841-64d847a2452a)

**Comment by LuluW8071** [comment]
Just look for research paper and read it or see documentation on pytorch official website. 

And I suggest you to read **Batch Normalization** paper as well. It was one of the revolutionizing paper to introduce solutions to faster converge while training and solve exploding/vanishing gradient problems.
